### PR TITLE
Description for Mac OS Parallels Desktop installation added

### PR DESCRIPTION
**Pull request by StephanStS**
Also Buster -> Bullseye  changed in the remaining text (but no new screenshots were generated!)

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -603,7 +697,7 @@ Select the following tabs for the installation description of your target.
 
         <font size="+2">1. Download and extract the DietPi direct write image</font>
 
-        Download the **BIOS direct write image** `DietPi_NativePC-BIOS-x86_64-Buster.7z` from [`dietpi.com`](https://dietpi.com/#download) and
+        Download the **BIOS direct write image** `DietPi_NativePC-BIOS-x86_64-Bullseye.7z` from [`dietpi.com`](https://dietpi.com/#download) and
```
Comment (MichaIng) [inline docs/install.md:700]: Probably we should remove the file name from here (and above cases) and replace it with what one actually sees at the linked download page:
```
Download the **Native PC (BIOS/CSM)** image from...
```
That way we do not need to remember updating those file names 😉.

File: docs/install.md [hunk 2]
```diff
@@ -406,6 +406,100 @@ Select the following tabs for the installation description of your target.
     <font size="+1">**Generate a Hyper-V Generation 2 machine**</font>  
     An option to get a Hyper-V Generation 2 machine is to generate your own Hyper-V image via a **Debian network installation** (booting the Hyper-V machine from a Debian `netinst.iso` installer like you would do it on a PC). Install a minimal Debian machine (i.e. no X11 desktops, etc.). Afterwards run the procedure described in section ["Make your own distribution"](../hardware/#make-your-own-distribution). Generation 2 machines support (and require) to boot in UEFI mode, support [Secure Boot](https://en.wikipedia.org/wiki/Unified_Extensible_Firmware_Interface#Secure_Boot), [TPM](https://en.wikipedia.org/wiki/Trusted_Platform_Module), use modern SCSI controllers and have higher hardware limits. For use as a home server, however, you will not need any of these functions.
 
+=== "Install in Parallels (Mac OS)"
```
Comment (MichaIng) [inline docs/install.md:409]: Not directly related to this PR, but what do you think about shortening the tab names, i.e. `Parallels (macOS)`, `VMware`, `VirtualBox`, skipping the `Install in/on`? Would enhance overview on small screens, also or especially once we add the scrollable tab style.

Comment (StephanStS) [reply docs/install.md:409]: Ok. Done.

File: docs/install.md (hunk 1 above)
Comment (StephanStS) [reply docs/install.md:700]: Ok. Done.

**Pull request merged**